config.yaml

```diff
@@ -2,7 +2,8 @@ DB_N: "/projects/ps-yeolab3/bay001/annotations/nr/nt"
 DB_X: "/projects/ps-yeolab3/bay001/annotations/nr/nr.dmnd"
 
 
-N_downsample_reads: 10000
+N_downsample_reads: 100000
 
 SAMPLESPATH:
-  - "/oasis/tscc/scratch/s5xu/share/LARP6.CTRL_IN1.umi.r1.fqdown.genome-mapped.bam"
+  - "/home/bay001/projects/adri_chimeric_clips_20220811/temporary_data/crispr_clip_cas711_se_GRCh38_v2_small/results/Cas711_SE_GRCh38.EEF1A1_22nt_Cas711_IP_r1.umi.r1.fq.genome-mapped.bam"
```

Comment:
If possible, let's keep the example config as-is, and point to small data/files that we've already created